assets/src/scss/layout/_navbar.scss

```diff
@@ -371,69 +208,106 @@ a.nav-link {
     transform: scaleX(0);
     transition: transform 0.25s;
     z-index: -1;
-
-    @include large-and-up {
-      border-bottom-width: 4px;
-    }
   }
 }
 
-.active a.nav-link {
-  --nav-link-active-- {
-    color: $white;
+// Underline animation with border-bottom
+// Using > * because there are some cases where
```

Comment:
Through accordion cases, `nav-link` is not the direct children of the `nav-item`